or-monte-carlo/src/com/waterloorocketry/openrocket_monte_carlo/SimulationData.java

```diff
@@ -7,165 +7,222 @@
 import info.openrocket.core.simulation.FlightDataType;
 import info.openrocket.core.simulation.FlightEvent;
 import info.openrocket.core.simulation.exception.SimulationException;
+import info.openrocket.core.unit.Unit;
+import info.openrocket.core.unit.UnitGroup;
+import info.openrocket.core.util.Chars;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
+import java.util.ArrayList;
 import java.util.Collections;
 import java.util.Comparator;
 import java.util.List;
 import java.util.Optional;
+import java.util.stream.Collectors;
 
 /**
  * Relevant data collected from run one of a simulation
  */
 public class SimulationData {
-    private final Simulation simulation;
+    private final static Logger log = LoggerFactory.getLogger(SimulationData.class);
+
+    private Simulation simulation;
+    private final String name;
     private double apogee;
-    private double minStability;
-    private double maxStability;
-    private double apogeeStability;
-    private double initStability;
+    private final List<String> branchName = new ArrayList<>();
```

Comment:
maybe it would be easier to work with if you just make a new SimulationData object for each flight branch?

Reply:
Perhaps, but we would still need to group by simulation when exporting/displaying data which could also get messy. Probably won't do such a large refactor with comp coming up.

Reply:
That's fair